backend/src/main/java/gov/cdc/usds/simplereport/validators/CsvValidatorUtils.java

```diff
@@ -193,7 +192,20 @@ public static List<FeedbackMessage> validateClia(ValueOrError input) {
   }
 
   public static List<FeedbackMessage> validateDate(ValueOrError input) {
-    return validateRegex(input, DATE_REGEX);
+    List<FeedbackMessage> errors = new ArrayList<>();
```

Comment:
@zdeveloper - Do you know if ReportStream accepts two-digit years? 11/3/80 instead of 11/3/1980? I'd like to change the patient bulk uploader to support two-digit years, but if that won't work for the results bulk uploader then I'll create a new validation method just for the patient uploader.
CC also @DanielSass @mehansen @nathancrtr

Reply:
I tested a two-digit year in my local, which points to staging, and RS didn't complain, so I think it's good
friendly reminder to update `frontend/src/app/testResults/uploads/schema.json` with any change to accepted format of data 😁

Reply:
Thank you! In the interest of keeping functionality changes from the patient uploader from bleeding into the results uploader I've just created a separate method for it, but it's available if y'all decide you want to allow short dates in the future :)